spaceship/lib/spaceship/portal/provisioning_profile.rb

```diff
@@ -260,16 +263,19 @@ def create!(name: nil, bundle_id: nil, certificate: nil, devices: [], mac: false
         #  If you're calling this from a subclass (like AdHoc), this will
         #  only return the profiles that are of this type
         # @param mac (Bool) (optional): Pass true to get all Mac provisioning profiles
-        # @param xcode DEPRECATED
+        # @param xcode (Bool) (optional): Pass true to include Xcode managed provisioning profiles
         def all(mac: false, xcode: false)
-          profiles = client.provisioning_profiles(mac: mac).map do |profile|
-            self.factory(profile)
+          if ENV['SPACESHIP_AVOID_XCODE_API']
```

Comment:
`spaceship` is still intended to avoid `fastlane_core` dependencies as far as I know. 👍